README.md

```diff
@@ -59,6 +59,11 @@ sudo NETWORK=testnet ./scripts/start
 sudo NETWORK=regtest ./scripts/start
 ```
 
+If you host Umbrel on a VPS (while it is **not recommended** as Umbrel is designed to be run on a secure local network), set the `UMBREL_BIND_ADDRESS` environment variable to `127.0.0.1`:
```

Comment:
I think it's best to not even mention VPS usage here as we really don't want to encourage it at this point.

We can just say if you don't want to open any ports and only want to expose Umbrel via Tor hidden services then set `UMBREL_BIND_ADDRESS=127.0.0.1`.

Reply:
Sure.

Reply:
Resolved in e3e25b8.